include/benchmark/benchmark.h

```diff
@@ -826,6 +822,9 @@ class Benchmark {
   static void AddRange(std::vector<int>* dst, int lo, int hi, int mult);
 
  private:
+  // prepare() is called before this benchmark is run.
+  virtual void prepare();
```

Comment:
This should be upper case as with the rest of the methods. I would vote for the name `Init()`

Reply:
I will change it to upper case, and use "Init()" if no others object. @dominichamon